docs(api): Flex revisions for Advanced Control

jwwojak opened the pull request:
# Overview

This is for revisions to the API doc, [Advanced Control](https://docs.opentrons.com/v2/new_advanced_running.html) (`new_advanced_running.rst`). Changes make this less OT-2 specific, incorporate Flex into the doc, and are very light.


# Test Plan

* Our usual review process.

* Sandbox:  http://sandbox.docs.opentrons.com/advanced-controls-revisions/v2/new_advanced_running.html

# Changelog

Changes include removing single OT-2 mentions and including both robots in text. Also, we're using generic terms for both like "robots".

Some minor text changes.

# Review requests

No specific requests.

# Risk assessment

Low. Changes are trivial.

jwwojak committed:
Make robot references generic for both models

Say either "Flex and OT-2" or just "the robots".

ecormany commented on api/docs/v2/new_advanced_running.rst:
```diff
@@ -84,8 +84,9 @@ Creating the dummy protocol requires you to:
+    :substitutions:
 
-    requirements = {'apiLevel': '|apiLevel|', 'robotType': 'Flex'}
+    requirements = {'robotType': 'Flex','apiLevel': '|apiLevel|'}
```
```suggestion
    requirements = {'robotType': 'Flex', 'apiLevel': '|apiLevel|'}
```
tiny style change, won't affect functionality

ecormany commented on api/docs/v2/new_advanced_running.rst:
```diff
@@ -84,8 +84,9 @@ Creating the dummy protocol requires you to:
+    requirements = {'robotType': 'Flex','apiLevel': '|apiLevel|'}
 
     def run(protocol):
         tips = protocol.load_labware('opentrons_flex_96_tiprack_50ul', 'D1')
```
Oops, I may have gotten a bit out over my skis here. Since Jupyter support is still forthcoming for 2.15, this is 🔮 code, not current reality.

In the meantime, we could revert this to the example from the previous iteration (updating the API level to 2.13):
https://github.com/Opentrons/opentrons/blob/a7a198e3254c6372cf3be8014c578027c6e438de/api/docs/v2/robot_position.rst?plain=1#L133-L141

jwwojak replied:
I recall seeing this while going through the doc:

>  requirements = {'robotType': 'Flex', 'apiLevel': '|apiLevel|'}

and thought, "hmmm, with 2.15, this isn't going to work given the note about versions" but just went with it anyway. Will reset to 2.13 and change the code to match an OT-2.

ecormany commented on api/docs/v2/new_advanced_running.rst:
```diff
@@ -99,13 +100,13 @@ After importing this protocol to the Opentrons App, run Labware Position Check t
 .. code-block:: python
 	
-    labware_1 = protocol.load_labware("opentrons_flex_96_tiprack_200ul", location="D1")
+    labware_1 = protocol.load_labware('opentrons_flex_96_tiprack_200ul', location='D1')
```
…and here we should match what we do above. if we want to match the previous (OT-2, 2.13) revision:

https://github.com/Opentrons/opentrons/blob/a7a198e3254c6372cf3be8014c578027c6e438de/api/docs/v2/robot_position.rst?plain=1#L147-L154

either way, we should probably keep the double quotes instead of single quotes — yes, they don't match the other code we wrote for this page, but this is _autogenerated_ and is meant to represent exactly what you would get from the Get Labware Offset Data feature in the app.

jwwojak replied:
Fixing to match output.

ecormany reviewed (approved):
Looks good. See comments about maybe rolling the example back to OT-2 / API version 2.13 for now.

jwwojak committed:
Apply suggestions from code review

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Fixing examples and quote marks

Changed the protocol example to OT-2 to match the API level 2.13.  Remove ' ' in favor of " "

jwwojak committed:
Merge branch 'edge' into advanced-controls-revisions
